src/assets/config/changelog.json

```diff
@@ -1,21 +1,9 @@
 {
   "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.20.md#v22012",
   "entries": [
-    {
-        "category": "changed",
-        "description": "Stopped overriding upstream chart tolerations for logging/promtail by default. Added `node-role.kubernetes.io/control-plane` toleration."
-    },
-    {
-        "category": "fixed",
-        "description": "Fixed an issue with wrong Corefile in NodeLocal DNSCache mounts. Added support for a kube-dns configmap for NodeLocal DNSCache to allow customization of dns."
-    },
-    {
-        "category": "fixed",
-        "description": "Fixed an issue where creating Clusters through ClusterTemplates was failing without leaving any trace."
-    },
     {
         "category": "fixed",
-        "description": "Fixed an API error in extended disk configuration for the Anexia provider."
+        "description": "Update machine-controller to version v1.54.1 to fix container runtime setup for new nodes"
     } 
```

Comment:
Can you please also add an entry for https://github.com/kubermatic/kubermatic/pull/11750? That's a user-facing change in 2.20.13.

Reply:
Good point. I run gchl only for dashboard, forgot about kubermatic repo.